src/patternfly/components/InlineEdit/inline-edit.scss

```diff
@@ -0,0 +1,202 @@
+// Don't remove this magic comment. See gulpfile.js.
+// @import "../../sass-utilities/all";
+
+.pf-c-inline-edit {
+  // Value group
+  --pf-c-inline-edit__value-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Input group
+  --pf-c-inline-edit__input-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Toggle group
+  --pf-c-inline-edit__title-group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Action
+  --pf-c-inline-edit__action-group--item--MarginRight: var(--pf-global--spacer--md);
+  --pf-c-inline-edit__action-group--m-inline--item--MarginRight: 0;
+
+  // Toggle
+  --pf-c-inline-edit--m-active__toggle--Color: var(--pf-global--Color--100);
+
+  // Save button
+  --pf-c-inline-edit__action--m-save--m-valid--c-button--m-plain--Color: var(--pf-global--link--Color);
+
+  // Hidden
+  .pf-m-hidden {
+    display: none;
+    visibility: hidden;
+  }
+
+  // Align right
+  .pf-m-align-right {
+    margin-left: auto;
+  }
+
+  // Overrides
+  // table {
+  //   table-layout: fixed;
+  // }
+
+  td {
+    word-break: normal;
+  }
+
+  /* stylelint-disable selector-max-compound-selectors */
+  [class*="pf-m-grid"] [data-label] {
+    .pf-c-check,
+    .pf-c-radio {
+      input {
+        grid-column: 1;
+      }
+    }
+  }
+  /* stylelint-enable */
+}
+
+// Value group
+.pf-c-inline-edit__value-group {
```

Comment:
What's the purpose to splitting up `.pf-c-inline-edit__value-group` and `.pf-c-inline-edit__input-group`, I'm wondering if you can combine these containers to `pf-c-inline-edit__group` and then use modifiers on the input vs value groups. This way when content becomes editable, a class/prop could be added to the container rather than a whole container switch (same with the container for the title) But let me know what you think

Reply:
Yeah, I had thought about using a standard `__group` with modifiers. @mcoker wyt?